wdio/specs/homepage.js

```diff
@@ -117,4 +117,25 @@ describe('Municipality homepage', () => {
     const target = browser.getAttribute('.item.action.homepage-teaser:nth-child(4) .content .header a', 'target');
     assert(target == '_blank');
   });
+
+  it('should show QAs only for the chosen user type "residents" and the hebrew language', () => {
+    browser.url('/municipality-1/node/1?user_type=residents&language=he');
+    browser.waitForVisible('div=כמה סייעות יש בגן ילדים?');
```

Comment:
This has changed in the new design, it's not a `div` anymore, also this test is not checking that those are the only FAQs visible, it's only checking what's visible, if some FAQs from `business` user type are showing on the page we will not know.